### PR TITLE
LEAF-3568 - Copy request_service does not show on non admin

### PR DESCRIPTION
**Pull request by shaneodd**
When copying a request, services were not showing when a user was not an admin, this was due to the services call being an admin only call. I did not see any reason for this since none of the other sections working with services in this way needed to be admin only.

### CODE REVIEW

**Review by maxrichardjr — APPROVED** [review]

**Review by mgaoVA — CHANGES_REQUESTED** [review]
All functions referenced in the POST and DELETE sections will need an admin check.

**Review by pete-nerantzinis — APPROVED** [review]

**Review by raphaelfretter — APPROVED** [review]

**Comment by shaneodd** [comment]
> All functions referenced in the POST and DELETE sections will need an admin check.

I added in logic for the post and delete to have those checks in place. My initial change would have affected this.

**Review by maxrichardjr — APPROVED** [review]
Appears to be correct from my perspective

**Review by aerinkayne — APPROVED** [review]

**Review by Pelentan — CHANGES_REQUESTED** [review]

**Review by Pelentan — APPROVED** [review]

File: LEAF_Request_Portal/api/controllers/ServiceController.php [hunk 1]
```diff
@@ -57,18 +57,21 @@ public function post($act)
         $login = $this->login;
         $service = $this->service;
 
-        $this->verifyAdminReferrer();
+        if ($login->checkGroup(1)) {
```
[the remark this answers is not in the document]
Comment (shaneodd) [reply LEAF_Request_Portal/api/controllers/ServiceController.php:60]: It returns true if the group is there, null if it is not there. This is used across the site and would probably be better served to update the checkGroup function so return false to fully fulfil how it should be. We had a quick discussion and determined this was a code grooming item for a later time.

**Review by AmandaSumner — APPROVED** [review]

**Pull request merged**